packages/expo-calendar/ios/EXCalendar/EXCalendar.m

```diff
@@ -285,8 +283,15 @@ - (EKEventStore *)eventStore
              [NSString stringWithFormat:@"Calendar with id %@ is not of type `event`", calendarId],
              nil);
         return;
-      }
+      } 
+      
+      calendarEvent = [EKEvent eventWithEventStore:self.eventStore];
       calendarEvent.calendar = calendar;
+    } else {
+        reject(@"E_INVALID_CALENDAR_ID",
+             @"CalendarId is required.",
+             nil);
```

Comment:
Two things:
1) proper indentation
2.a) in general if method invocation isn't too long, we keep it as oneliner (`reject(@"E_INVALID_CALENDAR_ID", @"CalendarId is required.", nil);`)
2.b) another thing is to keep an eye on parameters alignment when method invocation is multiliner - keep all arguments aligned to the first one
```obj-c
       |
reject(@"E_INVALID_CALENDAR_ID",
       @"CalendarId is required.",
       nil);
       |
       ^
       aligning line 
```